cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -1228,7 +1276,7 @@ def _has_valid_raw(self, force: bool = False) -> bool:
             if is_visium_and_is_single_true and x.shape[0] != self.visium_and_is_single_true_matrix_size:
                 self._raw_layer_exists = False
                 self.errors.append(
-                    f"When {ERROR_SUFFIX_VISIUM_AND_IS_SINGLE_TRUE}, the raw matrix must be the "
+                    f"When {self._visium_error_suffix}, the raw matrix must be the "
```

Comment:
This requirement is only enforced when is single is True, and ERROR_SUFFIX_VISIUM /  ERROR_SUFFIX_VISIUM_11M don't capture that. We should either make ERROR_SUFFIX_VISIUM_11M_AND_IS_SINGLE_TRUE or just hardcode the 'is_single' part of the requirement here (if cleaner that way)

Reply:
yep. Added a `ERROR_SUFFIX_IS_SINGLE` term and included in error message (and validator).